source/features/code-highlight.js

```diff
@@ -0,0 +1,43 @@
+import prism from "prismjs";
+import 'prismjs/components/prism-jsx'
+import 'prismjs/components/prism-bash'
+import 'prismjs/components/prism-git'
+import 'prismjs/components/prism-typescript'
+import 'prismjs/components/prism-scss'
+
+function isSupportedLanguage(lang) {
+	const supportedLang = ["javascript", "jsx", "bash", "git", "typescript", "html", "css", "scss"]
```

Comment:
Just make this variable top-level, so it's not recreated on each call and make it a `Set`.

Reply:
What's the advantage of using a `Set` over an `Array`? (just curious)

Reply:
It's more correct to use a Set for unordered collections with unique items. And it's also faster, but that shouldn't matter here.

Reply:
Nice to know, thank you!